dodola/tests/test_services.py

```diff
@@ -13,14 +14,15 @@ def _datafactory(x, start_time="1950-01-01"):
     if x.ndim != 1:
         raise ValueError("'x' needs dim of one")
 
+    time = pd.date_range(start=start_time, freq="D", periods=len(x))
```

Comment:
We can definitely switch back to using `xr.cftime_range`, I had switched to `pd.date_range` just for testing. I did use a standard calendar though here since we want to support leap days in our downscaling workflow, so probably makes sense to test here with them too.